scripts/imgtool/image.py

```diff
@@ -97,12 +97,15 @@ def __init__(self, endian, magic=TLV_INFO_MAGIC):
     def __len__(self):
         return TLV_INFO_SIZE + len(self.buf)
 
-    def add(self, kind, payload):
+    def add(self, kind, payload, is_custom=False):
```

Comment:
I'm more inclined to just have the callers lookup the kinds, pass the numeric kind values into this function, and not need extra complexity of the `is_custom` argument.